access api and capture data

RHunjan opened the pull request:
Accessed the api and captured data

RHunjan committed:
access api and capture data

RHunjan committed:
create coordinate array and station cards with station details

RHunjan committed:
clear search and add error message

RHunjan commented:
Hello All,

The ev api has been added.
* The coordinates for the map api are in mapArray
* I've added a div to the HTML with id="station-detail" where 5 cards are appended with the info for the stations. These will need to be styled with css.
* We are not supposed to use alerts, so the CSS team needs to build a modal for the error message
* We have to decide what data we want to store in local storage

Looking forward to our touch base tomorrow!

NewChap2022 commented:
> Hello All,
> 
> The ev api has been added.
> 
> * The coordinates for the map api are in mapArray
> * I've added a div to the HTML with id="station-detail" where 5 cards are appended with the info for the stations. These will need to be styled with css.
> * We are not supposed to use alerts, so the CSS team needs to build a modal for the error message
> * We have to decide what data we want to store in local storage
> 
> Looking forward to our touch base tomorrow!

Hi, Reena. 

Because your file is not the most updated one. I have added CSS style and changed some html already. Therefore, lots of code is not working now. I will see what I can do to fix it.

Thanks,

Shu

NewChap2022 commented:
I fixed the problem and committed the change to feature/css-styling branch to avoid any problem and will work on modal tonight.

RHunjan committed:
added local storage for last city and updated input placeholder

RHunjan committed:
local storage

RHunjan committed:
local storage for last city and input text

RHunjan commented:
Ok! Had a bit of trouble figuring how to get the latest files, but believe it worked out.

Added local storage for the last city and changed text in input box so users can enter street address, city or postal code. 

The error modal is poping up with entering nothing or entering an invalid address - so that's all good :)

rfabreu committed:
Merge branch 'develop' into feature/ev-api
